<launch>
	<arg name="copter_prefix" default="copter" />
	<arg name="copter_id" default="1" />
	<node pkg="burrito" type="safe_takeoff.py" name="safe_takeoff" ns="/hellacopters" output="screen">
	</node>
</launch>